packages/components/ovh-shell/src/client/index.ts

```diff
@@ -1,8 +1,10 @@
 import ShellClient from './shell-client';
 import exposeApi from './api';
 
-export default function useShellApi() {
+export function useShellApi() {
```

Comment:
Any reason of not using arrow functions?

Reply:
I don't think it changes anything here, the function relies on its own context so using arrow functions doesn't have any added value

Reply:
yes i think it's a matter of preferences, i believe that using named regular function might facilitates debugging as you get the function name in the debugger but that's pretty much it